### DISCUSSION TITLE
Default path for jupyter-book CLI switched to "./" (rather than required argument)

### DISCUSSION THREAD

**Discussion by mmcky** [executablebooks/meta#131] [Ideas]
Currently the [jupyter-book cli](https://jupyterbook.org/reference/cli.html#jupyter-book-build) requires a PATH to be entered when building books. 

Is there a reason we can't set

```
PATH_SOURCE = "./" (default value)
```

with an Error if it can't find `_config.yml` or `_toc.yml` in the current working directory. 

From my perspective it would be nice to be able to issue:

`jb build` and not `jb build ./`

as this is similar to a lot of other `cli` tools such as `git` that are `context` aware.

**Comment by chrisjsewell** [comment]
https://github.com/executablebooks/jupyter-book/pull/890
